desktop-style.css

```diff
@@ -357,4 +357,33 @@
   .mobile {
     display: none;
   }
+
+  .popup-card {
+    padding: 2%;
+  }
+
+  .cloned-desk-img {
+    height: 40vh;
+    width: 80vw;
+  }
+
+  .pop-body-container {
+    display: flex;
+    flex-direction: row;
+    width: 85vw;
+    margin-top: 1vh;
+  }
+
+  .pop-info-container {
+    width: 35vw;
+  }
+
+  #cloned-tech li {
+    margin-top: 0.5vh;
+  }
+
+  .pop-description {
+    width: 45vw;
+    margin-right: 3vw;
+  }
```

Comment:
- [ ] I think here you just need to align the description with the technologies div so kindly fix it to match the design 🔥🔥🔥🙏🏻🙏🏻


- Your design
![image](https://user-images.githubusercontent.com/58553711/180113464-e2946172-0e54-48d8-a546-f1dbdc86dd64.png)



- Figma design
![image](https://user-images.githubusercontent.com/58553711/180113321-e3cec6d7-4045-43a6-a19a-f0b4d048a4b8.png)